server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuthenticateCountyAdministrator.java

```diff
@@ -94,16 +94,20 @@ protected String asmIdentity(final Request the_request) {
    */
   @Override
   public String endpoint(final Request the_request, final Response the_response) {
-    if (Authentication.authenticateAs(the_request, AdministratorType.COUNTY)) {
+    if (Main.authentication().
+        isAuthenticatedAs(the_request, 
+                          AdministratorType.COUNTY, 
+                          the_request.queryParams(AuthenticationInterface.USERNAME))) {
       ok(the_response, "Authenticated");
     } else {
       try {
-        final SubmittedUsernamePassword auth_info = 
-            Main.GSON.fromJson(the_request.body(), SubmittedUsernamePassword.class);
-        if (auth_info != null &&
-            Authentication.authenticateAs(the_request, auth_info, 
-                                          AdministratorType.COUNTY)) {
-          ok(the_response, "Authenticated");
+        if (Main.authentication().
+            authenticateAdministrator(the_request, the_response,
+                AdministratorType.COUNTY, 
+                the_request.queryParams(AuthenticationInterface.USERNAME),
```

Comment:
Why are you changing the style of submission from JSON request body to HTTP parameters? Is it to be able to present a relatively standard HTML form? I suspect @ranweiler would probably prefer a JSON request body, since that's what he had me change the old way to in the first place (though I kept the HTTP parameters around so we could use the static web page for testing).

Reply:
Can/should we support both, given the continued need for the static page for testing?

Reply:
I think it's a reasonable thing for us to continue to support.